packages/quantic/force-app/custom-token/test/classes/CustomTokenProviderTest.cls

```diff
@@ -0,0 +1,107 @@
+@isTest
+private class CustomTokenProviderTest {
+    static User createDummyUser(String alias, String profile) {
+      Profile p = [SELECT Id FROM Profile WHERE Name = :profile];
+
+      User user = new User(
+            Username = alias + '@user.org',
+            FirstName = 'Foo',
+            LastName = 'Bar',
+            Alias = alias.abbreviate(8),
+            Email = alias + '@user.org',
+            EmailEncodingKey = 'UTF-8',
+            LanguageLocaleKey = 'en_US',
+            LocaleSidKey = 'en_US',
+            ProfileId = p.Id,
+            TimeZoneSidKey = 'America/Los_Angeles'
+        );
+
+        insert user;
+
+        return user;
+    }
+
+    static TokenSettings__c setupTokenSettings(String platformUrl, String organizationId, String apiKey){
+        TokenSettings__c settings = TokenSettings__c.getOrgDefaults();
+        settings.PlatformUrl__c = platformUrl;
+        settings.OrganizationId__c = organizationId;
+        settings.ApiKey__c = apiKey;
+        
+        upsert settings;
+        return settings;
+    }
+    
+    static final String sampleHeadlessConfiguration = '{"platformUrl":"https://platform.cloud.coveo.com","accessToken":"xx564559b1-0045-48e1-953c-3addd1ee4457","organizationId":"searchuisamples"}';
```

Comment:
I assume this is a dummy API key ? 😄

Reply:
This a searchuisample public api key being used by the jsui and the ui-kit ;)